app/modules/invite/service.server.ts

```diff
@@ -154,10 +156,19 @@ export async function createInvite({
   const token = jwt.sign({ id: invite.id }, INVITE_TOKEN_SECRET, {
     expiresIn: `${INVITE_EXPIRY_TTL_DAYS}d`,
   }); //keep only needed data in token to maintain the size
+
   await sendEmail({
     to: inviteeEmail,
     subject: `You have been invited to ${invite.organization.name}`,
-    text: inviteEmailText({ invite, token }),
+    text: "",
```

Comment:
We should not remove that.  We still need the text email as some people have html emails disabled.